src/format_elf.rs

```diff
@@ -238,9 +238,9 @@ impl<'a> Elf<'a> {
                 _ => Cell::new(name),
             }
         };
-        for (i, phdr) in phdrs.into_iter().enumerate() {
-            let name_cell = ph_name_table(&phdr);
-            let flags = ph_flag(&phdr);
+        for (i, phdr) in phdrs.iter().enumerate() {
```

Comment:
Interesting, I thought this only landed in rust 2021? We should also update to 2021 edition

Reply:
> Interesting, I thought this only landed in rust 2021?

What landed in 2021 edition is that the intended behavior of `into_iter()` started to cover more types (more iterator types are now **actually** consumed, instead of referenced). Before that, `into_iter()` sometimes operated simply as `iter()` (referencing its iterator, instead of consuming it).

> We should also update to 2021 edition

I think migrating to edition 2021 is a good idea, unless you need to retain backward compatibility with old Rust compilers.